src/devices/Qmc5883l/FieldRange.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
You are missing the header in many files. Please add it:
```csharp
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

```

Please add it in all the cs files.